applications/FluidDynamicsApplication/custom_elements/fluid_element.h

```diff
@@ -371,6 +371,12 @@ class FluidElement : public Element
         TElementData& rData,
         MatrixType& rMassMatrix);
 
+    virtual void AddBoundaryIntegral(
+        TElementData& rData,
+        const Vector& rUnitNormal,
```

Comment:
What if we call this argument rNormal? (just in case one wants to work with the area normal).